### DISCUSSION TITLE
Some videos only have sound.

### DISCUSSION THREAD

**Discussion by Aria-97** [Serene-Arc/bulk-downloader-for-reddit#597] [Q&A] [answered]
Hi, I downloaded few small subreddits and at one sub around %70 of video files were corrupted. Both normal and files I checked are from v.reddit. I deleted them and started the download again but corrupted files stayed corrupted and rest were again OK. I tried to view them on the subreddit and both files were playing correctly. Videos I checked were around few seconds and few minutes.

Is that a known bug or is there any tweak to fix that? Otherwise the program is incredible thank you for your effort!

![image_2022-02-14_191516](https://user-images.githubusercontent.com/99672286/153902326-c4952f18-eac2-457e-82b5-f2de5787c357.png)

**Comment by Serene-Arc** [comment, marked as answer]
You need to install FFmpeg from their website [here](https://ffmpeg.org/download.html). The BDFR uses it for some modules to merge video and audio, so when it's missing, weird things happen with the files.

**Reply by Aria-97** [reply]
Ah thank you! Since some videos came ok I didnt thought problem would be that.

**Reply by Serene-Arc** [reply]
It depends on the source, only some modules require it. Other videos don't require it at all
